app/assets/stylesheets/variables/_amanita.scss

```diff
@@ -98,5 +123,123 @@ $DANGER_BG_COLOR:                   #FFE0E0;
 $DANGER_BORDER_COLOR:               #F07070;
 $DANGER_FG_COLOR:                   black;
 
-$WELL_BG_COLOR:                     rgba($pachycolea_light_stipe, 0.45);
+$BANNER_BORDER_WIDTH:               2px;
+$BANNER_BORDER_STYLE:               solid;
+$BANNER_BORDER_RADIUS:              5px;
+
+$ALERT_BORDER_WIDTH:                2px;
+$ALERT_BORDER_STYLE:                solid;
+$ALERT_BORDER_RADIUS:               5px;
+
+// --------------------------------------------------
+// Logo
+// --------------------------------------------------
+
+$LOGO_BORDER_COLOR:                 $velosa_light_veil;
+$LOGO_BORDER_WIDTH:                 1px;
+$LOGO_BORDER_STYLE:                 solid;
+$LOGO_BORDER_RADIUS:                0px;
+$LOGO_FG_COLOR:                     $muscaria_foreground;
+$LOGO_BG_COLOR:                     $muscaria_background;
+$LOGO_HOVER_FG_COLOR:               $muscaria_foreground;
+$LOGO_HOVER_BG_COLOR:               $muscaria_background;
+
+// --------------------------------------------------
+// Left Navigation Bar
+// --------------------------------------------------
+
+$LEFT_BAR_BORDER_COLOR:             $velosa_background;
+$LEFT_BAR_BORDER_WIDTH:             1px;
+$LEFT_BAR_BORDER_STYLE:             solid;
+$LEFT_BAR_BORDER_RADIUS:            5px;
+$LEFT_BAR_ITEM_BORDER_WIDTH:        1px;
+$LEFT_BAR_HEADER_FG_COLOR:          $muscaria_background;
+$LEFT_BAR_HEADER_BG_COLOR:          $velosa_background;
+$LEFT_BAR_FG_COLOR:                 $velosa_light_veil;
+$LEFT_BAR_BG_COLOR:                 $velosa_background;
+$LEFT_BAR_HOVER_FG_COLOR:           $velosa_light_veil;
+$LEFT_BAR_HOVER_BG_COLOR:           $muscaria_background;
+
+// --------------------------------------------------
+// Top Navigation Bar
+// --------------------------------------------------
+
+$TOP_BAR_BORDER_COLOR:              transparent;
+$TOP_BAR_BORDER_WIDTH:              1px;
+$TOP_BAR_BORDER_STYLE:              solid;
+$TOP_BAR_BORDER_RADIUS:             0;
+$TOP_BAR_FG_COLOR:                  $pachycolea_foreground;
+$TOP_BAR_BG_COLOR:                  $pachycolea_background;
+$TOP_BAR_LINK_FG_COLOR:             $pachycolea_light_stipe;
+$TOP_BAR_LINK_BG_COLOR:             transparent;
+$TOP_BAR_LINK_HOVER_FG_COLOR:       $pachycolea_light_stipe;
+$TOP_BAR_LINK_HOVER_BG_COLOR:       $muscaria_background;
+
+// --------------------------------------------------
+// Lists and Tables
+// --------------------------------------------------
+
+$LIST_BORDER_COLOR:                 transparent;
+$LIST_HEADER_FG_COLOR:              #777777;
+$LIST_HEADER_BG_COLOR:              #EEEEEE;
```

Comment:
These variables probably need attention in all themes. List headers should use theme colors.